### PR TITLE
Get kafkas

### PR DESCRIPTION
**Pull request by dimakis**
To verify run:

```shell
ansible-galaxy collection build --force
ansible-galaxy collection install <ANSIBLE-COLLECTION-NAME-FROM-PREVIOUS-STEP> -vvv --force
```

just make sure the path to the `tar.gz` file is accurate

Then one can run a playbook of this:

```yaml
---
- name: RHOAS test
  hosts: localhost
  gather_facts: false
  connection: local
  tasks:
  - name: Get kafkas
    rhoas.rhoas.get_kafkas:
      # openshift_offline_token
    register:
      kafkas_resp_obj
  - name: Debug kafkas
    debug:
      msg: "{{ kafkas_resp_obj }}"
```

Which should spit back all kafkas available to the users org

### CODE REVIEW

**Comment by jackdelahunt** [comment]
When running `ansible-galaxy collection build --force` I get the file `rhoas-rhoas-0.1.0-alpha.tar.gz` not `rhoas-rhoas-0.1.1.tar.gz`.

**Comment by dimakis** [comment]
> When running `ansible-galaxy collection build --force` I get the file `rhoas-rhoas-0.1.0-alpha.tar.gz` not `rhoas-rhoas-0.1.1.tar.gz`.

shouldn't make a difference. its just whatever version is in the galaxy.yml
does the playbook run successfully?

**Comment by jackdelahunt** [comment]
```
 app-services-ansible git:(e266083) ✗ ansible-galaxy collection rhoas-rhoas-0.1.0-alpha.tar.gz -vvv --force

usage: ansible-galaxy collection [-h] COLLECTION_ACTION ...
ansible-galaxy collection: error: argument COLLECTION_ACTION: invalid choice: 'rhoas-rhoas-0.1.0-alpha.tar.gz' (choose from 'download', 'init', 'build', 'publish', 'install', 'list', 'verify')

usage: ansible-galaxy [-h] [--version] [-v] TYPE ...

Perform various Role and Collection related operations.

positional arguments:
  TYPE
    collection   Manage an Ansible Galaxy collection.
    role         Manage an Ansible Galaxy role.

options:
  --version      show program's version number, config file location, configured module search path, module location, executable location and
                 exit
  -h, --help     show this help message and exit
  -v, --verbose  Causes Ansible to print more debug messages. Adding multiple -v will increase the verbosity, the builtin plugins currently
                 evaluate up to -vvvvvv. A reasonable level to start is -vvv, connection debugging might require -vvvv.
```

**Comment by dimakis** [comment]
> No not working right now
> 
> ```
>  app-services-ansible git:(e266083) ✗ ansible-galaxy collection rhoas-rhoas-0.1.0-alpha.tar.gz -vvv --force
> 
> usage: ansible-galaxy collection [-h] COLLECTION_ACTION ...
> ansible-galaxy collection: error: argument COLLECTION_ACTION: invalid choice: 'rhoas-rhoas-0.1.0-alpha.tar.gz' (choose from 'download', 'init', 'build', 'publish', 'install', 'list', 'verify')
> 
> usage: ansible-galaxy [-h] [--version] [-v] TYPE ...
> 
> Perform various Role and Collection related operations.
> 
> positional arguments:
>   TYPE
>     collection   Manage an Ansible Galaxy collection.
>     role         Manage an Ansible Galaxy role.
> 
> options:
>   --version      show program's version number, config file location, configured module search path, module location, executable location and
>                  exit
>   -h, --help     show this help message and exit
>   -v, --verbose  Causes Ansible to print more debug messages. Adding multiple -v will increase the verbosity, the builtin plugins currently
>                  evaluate up to -vvvvvv. A reasonable level to start is -vvv, connection debugging might require -vvvv.
> ```

I left out the `install` keyword. so the collection wasn't being installed. I've edited it and put it in

File: plugins/modules/delete_service_account_by_id.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 short_description: Delete a Service Account for use with Red Hat Openshift Application Services.
```
Comment (rkpattnaik780) [inline plugins/modules/delete_service_account_by_id.py:17]:
```suggestion
short_description: Delete a Service Account for use with Red Hat OpenShift Application Services.
```
We could do a quick search for other places where the same capitalization can be done.

**Review by rkpattnaik780 — COMMENTED** [review]
Great work! Leaving some minor suggestions in docs.

**Comment by dimakis** [comment]
@rkpattnaik780 great review and suggestions, thank you. are you ok to approve it now? Did the new functionality work for you?

**Comment by rkpattnaik780** [comment]
> @rkpattnaik780 great review and suggestions, thank you. are you ok to approve it now? Did the new functionality work for you?

Getting this issue: `ERROR! Invalid collection name 'rhoas-rhoas-0.1.1.tar.gz', name must be in the format <namespace>.<collection>.`

**Comment by jackdelahunt** [comment]
Worked for me LGTM 👍

**Pull request merged**